source/read/retrieve.txt

```diff
@@ -0,0 +1,229 @@
+.. _pymongo-retrieve:
+
+=============
+Retrieve Data
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code examples, read, search, cursor
+
+Overview
+--------
+
+In this guide, you can learn how to use {+driver-long+} to retrieve
+data from a MongoDB collection by using read operations. You can call the
+``find()`` or ``find_one()`` method to retrieve documents that match a set of criteria.
+
+.. TODO: .. tip:: Interactive Lab
+   You can complete a short interactive lesson that demonstrates how to
+   retrieve data by using the ``Find()`` method in an in-browser coding
+   experience. To learn more, see the :ref:`pymongo-retrieve-instruqt-lab`
+   section of this guide.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the :ref:`<pymongo-quickstart>`.
+
+.. _pymongo-retrieve-find:
+
+Find Documents
+--------------
+
+{+driver-short+} includes two methods for retrieving documents from a collection:
+``find_one()`` and ``find()``.
+These methods take a **query filter** and return one or more matching documents.
+A query filter is an object that specifies the documents you want to retrieve in
+your query. 
+
+.. TODO: To learn more about query filters, see :ref:`pymongo-specify-query`.
+
+.. _pymongo-retrieve-find-one:
+
+Find One Document
+~~~~~~~~~~~~~~~~~
+
+To find a single document in a collection, call the ``find_one()`` method and pass a query
+filter that specifies the criteria of the document you want to find.
+If more than one document matches the query
+filter, this method returns the *first* matching document from the retrieved
+results as a Python dictionary. If no documents match the query filter, the method returns
+``None``.
+
+.. tip::
+   
+   The ``find_one()`` method is useful when you know there's only one matching document,
+   or you're only interested in the first match.  
+
+The following examples use the ``find_one()`` method to find the first document where
+the ``"cuisine"`` field has the value ``"Bakery"``. Select the corresponding tab to see
+an example in the Python shell or a ``.py`` file. 
```

Comment:
Probably a bit nitpicky, but I noticed these are out of order from the tabs now that they're switched. (applies to all)

```suggestion
an example in a ``.py`` file or the Python shell. 
```